specification/sql/resource-manager/Microsoft.Sql/preview/2023-08-01-preview/LocationCapabilities.json

```diff
@@ -1539,6 +1550,14 @@
           "type": "boolean",
           "readOnly": true
         },
+        "supportedZones": {
```

Comment:
>{ [](http://example.com/codeflow?start=26&length=1)

Should this be renamed to something like "supportedZoneCapabilities" or similar?

Reply:
these are specific SQL DB terms